projects/models.py

```diff
@@ -52,6 +52,18 @@ def is_overdue(self):
         today = date.today()
         return today > self.due_date and self.status != Project.ProjectStatus.COMPLETED
 
+    def set_to_done(self):
+        "Sets all the features associated with it to done, returns false if no features"
+        Project.objects.filter(id=self.id).update(status=30)
+        has_feature = False
+        data_f = Feature.objects.all()
+        Feature.objects.filter(project=self.id).update(status=20)
+        for feat in data_f:
```

Comment:
Checking whether a Project has any Features can be done by checking the count of the related ManyToOne field, rather than looping through all Features in the database.

Using the example in Django's documentation, to check if a Reporter has any Articles, it would be:
`
reporter.article_set.count() > 0`